coherence-spring-data/src/test/java/com/oracle/coherence/spring/data/model/BookProjection.java

```diff
@@ -1,7 +1,14 @@
+/*
+ * Copyright (c) 2013, 2021, Oracle and/or its affiliates.
```

Comment:
I adjusted the copyrights recently.  We can remove the `2013, ` from these as these files were introduced in 2021 (confirmed with Dhiru)